azure_functions_worker/dispatcher.py

```diff
@@ -640,8 +642,10 @@ async def _handle__invocation_request(self, request):
                 http_request = await http_coordinator.get_http_request_async(
                     invocation_id)
 
-                await sync_http_request(http_request, invoc_request)
-                args[fi.trigger_metadata.get('param_name')] = http_request
+                trigger_arg_name = fi.trigger_metadata.get('param_name')
+                func_http_request = args[trigger_arg_name]
+                await sync_http_request(http_request, func_http_request)
```

Comment:
sync_http_request takes in an invoc_request. Won't this fail if it does not find an invoc request?

Reply:
the invocation_request aka httprequest object reaching this point will be set in 
![{050D99A6-EAE1-4080-8A28-335BC28B86AD}](https://github.com/user-attachments/assets/917c7b98-adda-4419-847a-3700be0d4bc9). if host sends empty data for httptrigger, it will fail fast before this code